src/enemy.asm

```diff
@@ -0,0 +1,442 @@
+;
+; Animation settings
+;
+.code
```

Comment:
why this goes into `.code`? seems like `.rodata` is a better place for this stuff